provisioner.go

```diff
@@ -356,6 +400,9 @@ func (p *LocalPathProvisioner) createHelperPod(action ActionType, cmdsForPath []
 						},
 					},
 					ImagePullPolicy: v1.PullIfNotPresent,
+					SecurityContext: &v1.SecurityContext{
```

Comment:
I know it's due to `losetup` but I am concerned that for the user doesn't need this feature, they have to run another privileged container. It's better if we can find another way, e.g. expand the configmap. I have the same concern on https://github.com/rancher/local-path-provisioner/pull/126 too.